### PR TITLE
PT-5134: Remove assets from the platform

### PR DESCRIPTION
**Pull request by akak1977**
## Description
Remove assets from the platform

### CODE REVIEW

File: docs/techniques/how-to-run-platform-on-azure-functions.md [hunk 1]
```diff
@@ -1,29 +1,6 @@
 Azure Functions lets you run your code in a serverless environment without having to first create a virtual machine (VM) or publish a web application. 
 In this article, you use command-line tools to deploy and run VC Platform on the serverless environment  of Azure Functions.
 
-
-# How it works
```
Comment (Vectorfield4) [inline docs/techniques/how-to-run-platform-on-azure-functions.md:5]: Should we delete these files?

File: src/VirtoCommerce.Platform.Web/appsettings.json [hunk 2]
```diff
@@ -205,54 +205,5 @@
                 "PatternUrl": "/images/pattern-live.svg"
             }
         ]
-    },
-    "DefaultMainMenuState": {
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/appsettings.json:209]: Should we remove these?

File: tests/VirtoCommerce.Platform.Tests/Validators/ValidatorTestsHelper.cs [hunk 3]
```diff
@@ -1,15 +0,0 @@
-namespace VirtoCommerce.Platform.Tests.Validators
-{
-    public class ValidatorTestsHelper
```
Comment (Vectorfield4) [inline tests/VirtoCommerce.Platform.Tests/Validators/ValidatorTestsHelper.cs:3]: Should we remove these?

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.Platform.Web/appsettings.json (hunk 2 above)
Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/appsettings.json:209]: It was in obsolete dev. There isn't now.

**Pull request merged**